velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -699,6 +699,14 @@ void PageReader::makeDecoder() {
               "DELTA_BINARY_PACKED decoder only supports INT32 and INT64");
       }
       break;
+    case Encoding::DELTA_BYTE_ARRAY:
+      if (parquetType == thrift::Type::BYTE_ARRAY) {
+        deltaByteArrDecoder_ =
+            std::make_unique<DeltaByteArrayDecoder>(pageData_);
+        break;
+      } else {
+        VELOX_UNSUPPORTED("DELTA_BYTE_ARRAY decoder only supports BYTE_ARRAY");
```

Comment:
nit: I think we can remove this and use the default:

Reply:
Fixed.